src/api/matrix.js

```diff
@@ -0,0 +1,37 @@
+import Http from '../http';
+import Navigator from '../navigator';
+
+class Matrix {
+  static async getMatrixDistances(routingOptions={}) {
```

Comment:
Should this just be in `src/api/routing.js`? Not sure we need a separate file for matrix

Reply:
ah yea i'll move it